modules/insights-operator-one-time-gather.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc
+
+
+
+[id="insights-operator-one-time-gather_{context}"]
+= Running an Insights Operator gather operation 
+
+You must run a gather operation to create an Insights Operator archive.
+
+.Prerequisites
+
+* You are logged in to {product-title} as `cluster-admin`.
+
+.Procedure
+
+. Create a file named `gather-job.yaml` using this template:
++
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/insights-operator/release-4.8/docs/gather-job.yaml[]
```

Comment:
Just want to check to see that you've gotten required approvals for this embedded file, as per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#embedding-an-external-file

Reply:
Yes I have. It was not my first choice, but this template updates regularly at the moment. If it ever solidifies, I hope to just embed it into the doc.